File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-Java/features/Number/GPL/Graph.java
package GPL;

// ***********************************************************************
   
public class Graph 
{
    // Executes Number Vertices
    public void run( Vertex s )
     {
       	System.out.println("Number");
        NumberVertices( );
        original( s );
    }

    public void NumberVertices( ) 
    {
        GraphSearch( new NumberWorkSpace( ) );
    }
}

File: projects/GPL-FH-CSharp/features/Connected/Graph.cs
//layer Connected;

// *****************************************************************
   
public class Graph //refines
{
    // Executes Connected Components
    public void run( Vertex s )
    {
	     	System.Console.Out.WriteLine("Connected");
        ConnectedComponents( );
        original( s );
    }

    public void ConnectedComponents( ) 
    {
        GraphSearch( new RegionWorkSpace( ) );
    }
}
